### PR TITLE
🌐 Add German translation for `docs/deployment/index.md`

### PR DESCRIPTION
**Pull request by mawassk**

### CODE REVIEW

File: docs/de/docs/deployment/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Deployment - Einführung
```
Comment (0x4Dark) [inline docs/de/docs/deployment/index.md:1]: The first one is a bit difficult. It is supposed to be translated, so I think "Bereitstellung" or "Verteilung" would be more appropriate. However, deployment is also commonly used in German.

File: docs/de/docs/deployment/index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Das Bereitstellen einer **FastAPI** Anwendung ist relativ einfach.
+
+Hierbei gibt es abhängig von der Situation und den eingesetzen Technologien mehrere Möglichkeiten.
+
+In den nächsten Kapiteln befinden sich hierzu mehrere nähere Informationen und einige der möglichen Technologien werden vorgestellt.
```
Comment (0x4Dark) [inline docs/de/docs/deployment/index.md:7]: Alternative suggestion: 
```In den nächsten Abschnitten erfahren Sie mehr über die Details, die Sie beachten sollten, sowie über einige der Techniken, die Sie dabei anwenden können.```

File: docs/de/docs/deployment/index.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Deployment - Einführung
+
+Das Bereitstellen einer **FastAPI** Anwendung ist relativ einfach.
+
+Hierbei gibt es abhängig von der Situation und den eingesetzen Technologien mehrere Möglichkeiten.
```
Comment (0x4Dark) [inline docs/de/docs/deployment/index.md:5]: Maybe `Werkzeuge` ?

File: docs/de/docs/deployment/index.md (hunk 1 above)
Comment (mawassk) [reply docs/de/docs/deployment/index.md:1]: Yes, "Bereitstellung" is a good translation for deployment. But personally, in my opinion (native german) "Bereitstellung" seems to be more irritating than "Deployment". Maybe there is another reviewer, who want to share his opinion?

File: docs/de/docs/deployment/index.md (hunk 2 above)
Comment (mawassk) [reply docs/de/docs/deployment/index.md:7]: This seems to be better, but i would say: `In den nächsten Abschnitten erfahren Sie mehr über wichtige Details, sowie über einige der Techniken, die Sie dabei anwenden können.`

File: docs/de/docs/deployment/index.md (hunk 3 above)
Comment (mawassk) [reply docs/de/docs/deployment/index.md:5]: Isn´t `Werkzeuge` in this context to specific?

Comment (0x4Dark) [reply docs/de/docs/deployment/index.md:5]: For me, deployment is often about continuous deployment, so I'm thinking more along the tools spectrum. However, I have taken a closer look at the next chapters. It's less about the tools used, but more about the techniques behind them. Literally, "Werkzeuge" are more appropriate, but here it's more about the context. That's why I think "eingesetzen Technologien" is more appropriate.

**Review by 0x4Dark — APPROVED** [review]

**Review by rogerbrinkmann — COMMENTED** [review]
@mawassk , are you still proceeding with this? I'd be happy to review. But it seems the englisch version has already changed a lot.

**Comment by mawassk** [comment]
> @mawassk , are you still proceeding with this? I'd be happy to review. But it seems the englisch version has already changed a lot.

Due to the big changes in the english version i will close this PR. It seems to be better to work on this in another PR

**Pull request closed without merging**